core/src/main/java/org/elasticsearch/search/aggregations/AbstractAggregationBuilder.java

```diff
@@ -116,6 +116,11 @@ public AB setMetaData(Map<String, Object> metaData) {
     }
 
     @Override
+    public Map<String, Object> getMetaData() {
```

Comment:
I was trying to understand if there is a specific reason already to do this. My understanding og "metaData" in aggegation is that it is something that is piggy backed from the request to the response and currently is mostly used by our own clients, so it shouldn't change once it has been set once. Since this it the aggregation builder it might be okay to allow this at this point. Maybe @colings86 has an opinion on this?